tests/fix-stylish-haskell.nix

```
@@ -0,0 +1,20 @@
+{ writeScriptBin, git, fd, stylish-haskell }:
+
+writeScriptBin "fix-stylish-haskell" ''
+    ${git}/bin/git diff > pre-stylish.diff
+    ${fd}/bin/fd \
+      --extension hs \
+      --exclude '*/dist/*' \
+      --exclude '*/docs/*' \
```

```suggestion
    ${fd}/bin/fd --extension hs "$@" \
```
This would be a cheap and cheerful way to set the ignore list.
But the existing excludes would be fine for cardano-wallet at least.